docs/using-jda/interactions.md

```diff
@@ -423,29 +423,58 @@ Each non-link button requires such an ID in order to be used.
 
 ### Select Menus (Dropdowns)
 
-Select Menus can be disabled and have up to 25 options.
+Select Menus can be disabled.
 
 It's possible to set the minimum and maximum number of options to be selected.
 
+There are two implementations for SelectMenus:
+
+- [`StringSelectMenu`](#string-select-menus) supports custom string choices
+- [`EntitySelectMenu`](#entity-select-menus) derives its choices from mentionable entities (such as `User`s, `Channel`s, etc)
+
+#### String Select Menus
+
+String Select Menus support up to 25 options.
+
 Each option can have its own label, description, and emoji.
 There can be multiple options selected and set as default.
 
 ![Example Select Menu With A Default Value](https://i.imgur.com/44q006n.png)
 
-#### Handling SelectMenuInteractionEvent
+#### Entity Select Menus
 
-When a user selects their options from a dropdown and submits their choices, you will receive a `SelectMenuInteractionEvent` for the respective interaction with the selected values.
+Entity Select Menus do not support custom choices. Instead, they derive their choices from mentionable Discord entities
+such as `User`s, `Channel`s, etc.
 
-!!! example
+![Example Entity Select Menu for Roles](../assets/images/entity_select_example.png)
+
+You can specify which entity types you wish to appear as choices by specifying the `SelectTarget`s in `EntitySelectMenu.create`.
```

Comment:
Should this note that you cannot combine channel with role/user types?

Reply:
I think it'd be a good idea to mention said fact